url-gen/src/main/kotlin/com/cloudinary/transformation/delivery/DeliveryActions.kt

```diff
@@ -44,17 +44,22 @@ class DeliveryFormat(
     private val format: Format,
     private val lossy: Boolean? = null,
     private val progressive: Progressive? = null,
-    private val preserveTransparency: Boolean? = null
+    private val preserveTransparency: Boolean? = null,
+    private var ignoreMaskChannels: Boolean = false
+
 
 ) : Delivery() {
     override fun toString(): String {
         val lossyStr = if (lossy == true) "fl_lossy" else null
         val preserveTransparencyStr = if (preserveTransparency == true) "fl_preserve_transparency" else null
         val progressiveStr = progressive?.toString()
+        val ignoreMaskChannels = if(ignoreMaskChannels) "fl_ignore_mask_channels" else null
 
-        return "f_$format".joinWithValues(lossyStr, preserveTransparencyStr, progressiveStr, separator = ",")
+        return "f_$format".joinWithValues(lossyStr, preserveTransparencyStr, progressiveStr, ignoreMaskChannels, separator = ",")
     }
 
+    fun ignoreMaskChannels() = apply { this.ignoreMaskChannels = true }
```

Comment:
other functions are defined inside `class Builder` below, any reason for this one to be here?

It looks like `ignoreMaskChannels` is missing in the builder.